circle.yml

```diff
@@ -0,0 +1,34 @@
+machine:
+  php:
+    version: 5.6.22
+  environment:
+    WP_TESTS_DIR: /tmp/wordpress-tests-lib
+    WP_CORE_DIR: /tmp/wordpress/
+    PATH: $HOME/.composer/vendor/bin:$PATH
+
+dependencies:
+  pre:
+    - sudo apt-get update; sudo apt-get install subversion
+
+test:
+  pre:
+    - |
+      composer global require wp-coding-standards/wpcs
+      phpcs --config-set installed_paths $HOME/.composer/vendor/wp-coding-standards/wpcs
+  override:
+    - phpcs --standard=phpcs.ruleset.xml $(find . -name '*.php')
+    - |
+      rm -rf $WP_TESTS_DIR $WP_CORE_DIR
+      bash bin/install-wp-tests.sh wordpress_test ubuntu '' 127.0.0.1 4.6.4
```

Comment:
4.6.5 was released yesterday. Let's just update this now. We are always going to just be mindful of the latest release number while we are a version behind though.